src/internal/storage/fileset/util.go

```diff
@@ -144,3 +145,14 @@ func (i *Iterator) Next() (File, error) {
 		return nil, err
 	}
 }
+
+func hashDataRefs(dataRefs []*chunk.DataRef) ([]byte, error) {
```

Comment:
I did like having the public function `StableHash` is that gone now?

Reply:
Yeah, it is replaced by the `Hash` function on the fileset.File interface which provides a stable hash.